packages/volto/src/components/theme/AlternateHrefLangs/AlternateHrefLangs.jsx

```diff
@@ -0,0 +1,23 @@
+import config from '@plone/volto/registry';
+import Helmet from '@plone/volto/helpers/Helmet/Helmet';
+
+const AlternateHrefLangs = (props) => {
+  const { content } = props;
+  return (
+    <Helmet>
+      {config.settings.isMultilingual &&
+        content['@components']?.translations?.items.map((item, key) => {
+          return (
+            <link
+              key={key}
+              rel="alternate"
+              hreflang={item.language}
```

Comment:
@erral It does work on the SSR but I see this warning in client 
```
Warning: Invalid DOM property `hreflang`. Did you mean `hrefLang`?
    at link
    at head
    at html
    at Html     
```

 if we use hrefLang, I think the browser will handle the conversion. On the server we get it as `hrefLang`, does it cause issues in SEO ? I think the HTML attributes are case insensitive, the search engines should parse it as hreflang.

Reply:
> @erral It does work on the SSR but I see this warning in client
> 
> ```
> Warning: Invalid DOM property `hreflang`. Did you mean `hrefLang`?
>     at link
>     at head
>     at html
>     at Html     
> ```
> 
> if we use hrefLang, I think the browser will handle the conversion. On the server we get it as `hrefLang`, does it cause issues in SEO ? I think the HTML attributes are case insensitive, the search engines should parse it as hreflang.

mmmm, this may be something similar to `class` vs `className` when rendering the component.

I need to check this with SSR and the failing tests also, I will tackle this when I have some time later this week.

Reply:
@nileshgulia1 @erral HTML attributes are not case sensitive, but in React they are!

Reply:
yes ofc :)